runatlantis.io/docs/webhook-secrets.md

```diff
@@ -30,7 +30,7 @@ You can use any random string generator to create your Webhook secret. It should
 
 For example:
 * Generate via Ruby with `ruby -rsecurerandom -e 'puts SecureRandom.hex(32)'`
-* Generate online with [https://www.browserling.com/tools/random-string](https://www.browserling.com/tools/random-string)
+* Generate online or using your prefered password manager
```

Comment:
looks like it is accessible?
```
$ curl -I https://www.browserling.com/tools/random-string
HTTP/2 200
server: nginx
date: Wed, 05 Jul 2023 21:48:08 GMT
content-type: text/html; charset=UTF-8
```

Reply:
it was not earlier and to be honest, is not needed at all, one google search and you find millions of generators.
one less link to mantain.

Reply:
I will prefer to remove links that are not really crucial for the docs to make sense.

Reply:
yeah, that makes sense, random password generation is not our thing :D